addons/website_slides/static/src/js/slides_course_quiz.js

```diff
@@ -298,7 +312,19 @@
                 QWeb.render('slide.slide.quiz.validation', {'widget': this})
             );
         },
-
+        /*
+        * @private
+        * toggle additional resource info box
+        */
+        _toggleAdditionalResourceInfo: function(on) {
+            const resourceInfo = document.getElementsByClassName('o_wslides_js_lesson_quiz_resourceinfo')[0];
+            if(on){
+                resourceInfo.classList.remove('d-none');
+            }
+            else{
+                resourceInfo.classList.add('d-none');
+            }
```

Comment:
Styling guidelines-wise,

```suggestion
            if (show) {
                resourceInfo.classList.remove('d-none');
            } else {
                resourceInfo.classList.add('d-none');
            }
```

But would this work ?
```suggestion
            show ? resourceInfo.classList.add('d-none') : resourceInfo.classList.remove('d-none');
```
Also, `show` is more explicit than `on`? (I know I first named it so before)